ota_pal.c

```diff
@@ -73,14 +94,166 @@ static psa_image_id_t xOTAImageID = TFM_FWU_INVALID_IMAGE_ID;
 /* The key handle for OTA image verification. The key should be provisioned
  * before starting an OTA process by the user.
  */
-extern psa_key_handle_t xOTACodeVerifyKeyHandle;
+static psa_key_handle_t xOTACodeVerifyKeyHandle = ( psa_key_handle_t ) OTA_SIGNING_KEY_ID;
+
+static uint8_t ucECDSARAWSignature[ ECDSA_SHA256_RAW_SIGNATURE_LENGTH ] = { 0 };
```

Comment:
The scope of ucECDSARAWSignature could be reduced.

Reply:
Declare it if OTA_PAL_CODE_SIGNING_ALGO is OTA_PAL_CODE_SIGNING_ECDSA.